scripts/version.sh

```diff
@@ -24,14 +24,15 @@ fi
 
 # We're building k3s against containerd 1.5 in go.mod because 1.6 has dependency
 # conflicts with Kubernetes, but we still need to bundle containerd 1.6.
-VERSION_CONTAINERD="v1.6.10-k3s1"
+VERSION_CONTAINERD="v1.6.10-k3s2"
```

Comment:
is this just for this release, or will it always be a 2?

Reply:
It'll be k3s2 until containerd 1.6.11 comes out and we do a new release.